docs/current/cli/979595-reference.md

```diff
@@ -214,6 +329,54 @@ Disable Dagger terminal output, but continue emitting program standard output:
 dagger --silent run go run ci.go 2>&1 | tee foo.out
 ```
 
+## dagger shell
+
+:::note
+This command is currently under development and is therefore hidden in the CLI.
+:::
+
+Open a shell in a container returned by a function. If no entrypoint is specified and the container doesn't have a default command, `sh` will be used.
+
+### Usage
+
+```shell
+dagger shell [function]
+```
+
+### Example
+
+Open a shell session in the container returned by the `debug` function:
+
+```shell
+dagger shell debug
+```
+
+## dagger up
+
+:::note
+This command is currently under development and is therefore hidden in the CLI.
+:::
+
+Start a service or a container returned by a function and expose its ports to the host.
+
+:::note
+In order for this to work, the service/container returned by the function must have the `Container.withExposedPort` field defining one or more exposed ports.
+:::
+
+### Usage
+
+```shell
+dagger up [function]
+```
+
+### Example
+
+Start the service returned by the `debug` function:
+
+```shell
+dagger up debug
```

Comment:
Examples of the flags like --native and --publish would be good

Reply:
Couldn't find --publish

Reply:
Possibly meant `--port`.

Reply:
Checked and `--port` is already documented, nothing to do here.